lib/index.js

```diff
@@ -1992,6 +1992,47 @@ addMethod('toCallback', function (cb) {
 });
 
 /**
+ * Returns the result of a stream to a nodejs-style callback function.
```

Comment:
Update docs.

You can just say the stream must emit at most one value or error and that the promise will resolve to that value when it is emitted, then link to `[toCallback](#toCallback)` for details.